Packs/NCSCCyberAsssessmentFramework/Playbooks/playbook-NCSC_CAF_Assessment.yml

```
@@ -0,0 +1,9342 @@
+contentitemexportablefields:
+  contentitemfields:
+    fromServerVersion: ""
+    itemVersion: ""
+    packID: 243a1c9d-2ab7-4999-820b-f23d6816c229
+    propagationLabels:
+    - all
+    toServerVersion: ""
+id: 2fd1efe9-6f59-42e5-8db2-43a8799898e8
```

The playbook id should be the same as the playbook name.
```suggestion
id: NCSC CAF Assessment
```